packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -9,7 +9,7 @@
     "vscode": "^1.49.3"
   },
   "dependencies": {
-    "@salesforce/core": "2.28.4",
+    "@salesforce/core": "^2.35.0",
```

Comment:
This is now the version specific the same as it is in SRD and apex-node to avoid version mismatches.

Reply:
Is there a way to ensure this is in sync with SDR and apex-node going forward?

Reply:
not that I'm aware of out of the box, but this is a fairly unique use case.  I don't totally understand at this point how the different deps are all interconnected and how this results in the version issue encountered here.    I'll create a ticket for tracking and figuring out the issue